_posts/0000-01-02-NwosaEmeka.md

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
It's bee a **Huge** pleasure learning with *git bot*

My achievement so far

Achievement | Status
---------------|-----------
Create Branch | Check
Create File | Check
Commit a file | Check